eng/pipelines/publish.yml

```diff
@@ -1,4 +1,5 @@
 parameters:
+  artifactsDir: $(Build.SourcesDirectory)/artifacts
```

Comment:
I don't think it affects functionality, but why is this a template parameter rather than a build variable? I don't see why anyone would change this when using this as a template.